physicalplan/scan/function.go

```diff
@@ -455,7 +455,7 @@ func linearRegression(samples []promql.Point, interceptTime int64) (slope, inter
 	return slope, intercept
 }
 
-func kahanSumInc(inc, sum, c float64) (newSum, newC float64) {
+func KahanSumInc(inc, sum, c float64) (newSum, newC float64) {
```

Comment:
Should I move this func somewhere outside or keep it here ?

Reply:
I think it is okay to expose it for now. If needed we can extract the function outside in the future